### PR TITLE
CRUD Generator: Add support for date-only fields

### PR DESCRIPTION
**Pull request by dkarnutsch**
- Added lastCheckedAt to Demo to highlight differences between Date and DateTime
- Use Date for availableSince (requires graphql-scalars)

---

## PR Checklist

-   [x] Verify if the change requires a changeset. See [CONTRIBUTING.md](https://github.com/vivid-planet/comet/blob/HEAD/CONTRIBUTING.md)
-   [x] Link to the respective task if one exists: https://vivid-planet.atlassian.net/browse/COM-882

### CODE REVIEW

**Review by nsams — APPROVED** [review]

File: demo/admin/src/products/generated/ProductForm.tsx [hunk 1]
```diff
@@ -223,11 +223,12 @@ export function ProductForm({ id }: FormProps): React.ReactElement {
                                 />
                             )}
                         </Field>
+
                         <DateField
                             variant="horizontal"
                             fullWidth
-                            name="availableSince"
-                            label={<FormattedMessage id="product.availableSince" defaultMessage="Available Since" />}
```
Comment (johnnyomair) [inline demo/admin/src/products/generated/ProductForm.tsx:230]: It's strange that the generator generates a grid column for availableSince, but no form field. Is this intentional?

File: .changeset/neat-kiwis-serve.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"@comet/cms-api": minor
+---
+
+API: Add date filter
```
Comment (johnnyomair) [inline .changeset/neat-kiwis-serve.md:5]:
```suggestion
---
"@comet/cms-admin": minor
"@comet/cms-api": minor
---

CRUD Generator: Add support for date-only fields
```

File: demo/admin/src/products/generated/ProductForm.tsx (hunk 1 above)
Comment (nsams) [reply demo/admin/src/products/generated/ProductForm.tsx:230]: I wouldn't invest too much time into the old admin-generator

**Comment by dkarnutsch** [comment]
@johnnyomair fixed and conflicts resolved.

As @nsams suggested, I did not make changes to the old generator. However, the new generator should support date-only fields. Should this be done in a new PR or do you prefer this to be done is this PR? Considering the size, I would prefer to do this in another PR.

**Comment by johnnyomair** [comment]
> Should this be done in a new PR or do you prefer this to be done is this PR?

You can add this in a follow-up PR

**Comment by johnnyomair** [comment]
Now that next has been merged into main, could you target main for this?

**Review by nsams — APPROVED** [review]

**Review by johnnyomair — APPROVED** [review]

**Pull request merged**